azurerm_cdn_endpoint/main.tf

```diff
@@ -23,6 +23,34 @@ resource "azurerm_cdn_endpoint" "cdn_endpoint" {
     host_name = var.origin_host_name
   }
 
+  dynamic "global_delivery_rule" {
+    iterator = l
+    for_each = var.global_delivery_rule_cache_expiration_action
```

Comment:
if you want to be sure the order of the rules never change even if you change them when you define their values you can do something like this:
https://github.com/pagopa/io-infrastructure-modules-new/blob/26e0149001372ee2d4b3a5ac6714ba074a352bc7/azurerm_application_gateway/main.tf#L188
But it's not mandatory

Reply:
Ok, I changed the code (see also comments below).